internal/scheduler/yunikorn/resourceusage/memory.go

```diff
@@ -103,6 +122,11 @@ func executorMemoryRequest(app *v1beta2.SparkApplication) (string, error) {
 		return "", err
 	}
 
+	pysparkMemoryBytes, err := executorPysparkMemoryBytes(app)
+	if err != nil {
+		return "", err
+	}
+
 	// See comment above in driver
-	return bytesToMi(requestBytes), nil
+	return bytesToMi(requestBytes + pysparkMemoryBytes), nil
```

Comment:
I think I'm also missing offheap bytes in this calculation https://github.com/apache/spark/blob/7de71a2ec78d985c2a045f13c1275101b126cec4/core/src/main/scala/org/apache/spark/resource/ResourceProfile.scala#L525-L532